javascript/ql/src/Security/CWE-078/IndirectCommandInjection.qhelp

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+<overview>
+
+	<p>
+
+		Forwarding command-line arguments to
+		<code>require('child_process').exec</code>, or some other library
+		routine that executes a system command can change the meaning of the
```

Comment:
Either add a comma after `command` or remove the comma before `or` on the previous line.